lib/Doctrine/ODM/MongoDB/SchemaManager.php

```diff
@@ -1079,6 +1080,10 @@ private function getWriteOptions(?int $maxTimeMs = null, ?WriteConcern $writeCon
 
     private function isSearchIndexCommandException(CommandException $e): bool
     {
+        if ($e->getCode() === self::CODE_SEARCH_NOT_ENABLED) {
```

Comment:
We can add a comment as other cases.
```suggestion
        // MongoDB 8.0+: "Using Atlas Search Database Commands and the $listSearchIndexes aggregation stage requires additional configuration."
        if ($e->getCode() === self::CODE_SEARCH_NOT_ENABLED) {
```

Reply:
I was thinking that, but I wasn't sure if it was version related or simply overlooked. Wasn't sure which repo to check to compare, maybe you can enlighten me?

Reply:
I tested all minor versions and community/enterprise/atlas when I wrote the code for Laravel. Trust me 😏